src/glossary.rst

```diff
@@ -1051,6 +1051,14 @@ Glossary
 
          * :term:`reinstall`
 
+   install target
+      Cylc uses install targets to determine which hosts share file systems,
+      these are expected to be managed at site level and configured in
```

Comment:
```suggestion
      Cylc uses install targets to determine which platforms share file systems.
      Install targets should normally be managed at site level: They are configured in

Reply:
Can we either replace the capital with lowercase or the colon with a full stop?

```suggestion
      Cylc uses install targets to determine which platforms share file systems.
      Install targets should normally be managed at site level. They are configured in
```